### PR TITLE
fix idioms to simple words

### PR DESCRIPTION
**Pull request by koyakei**
use 1st meaning in google mouse over dictionary.

### CODE REVIEW

File: en/intro.rst [hunk 1]
```diff
@@ -150,8 +150,7 @@ features in CakePHP are:
 * A :doc:`caching </core-libraries/caching>` framework that integrates with
   Memcache, Redis and other backends.
 * Powerful :doc:`code generation tools
-  </bake/usage>` so you can hit the ground
-  running.
+  </bake/usage>` so you can start immidiately.
```
Comment (markstory) [inline en/intro.rst]: 'immediately' is the correct spelling I think.

File: en/intro.rst [hunk 2]
```diff
@@ -18,7 +18,7 @@ CakePHP provides a basic organizational structure that covers class names,
 filenames, database table names, and other conventions. While the conventions
 take some time to learn, by following the conventions CakePHP provides you can
 avoid needless configuration and make a uniform application structure that makes
-working with various projects a breeze. The :doc:`conventions chapter
+working with various projects smoothly. The :doc:`conventions chapter
```
Comment (markstory) [inline en/intro.rst]: simple instead of smoothly?

Comment (koyakei) [inline en/intro.rst]: Do you mean, you think it should use "simple" instead of smoothly?

Comment (markstory) [inline en/intro.rst]: Yes.

**Comment by markstory** [comment]
Thanks:+1:

**Pull request merged**